src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -302,7 +302,7 @@ public PackageBuilder CreateBuilder(string basePath, NuGetVersion version, strin
             {
                 Logger.Log(
                     PackagingLogMessage.CreateWarning(
-                        string.Format(NuGetResources.ProjectJsonPack_Deprecated, builder.Id),
+                        string.Format(CultureInfo.CurrentCulture, NuGetResources.ProjectJsonPack_Deprecated, builder.Id),
```

Comment:
Should we be using `CurrentCulture` or `CurrentUICulture`?  I've never understood when to use which 😕

Reply:
I'm confused; dotnet uses DefaultThreadCurrentUICulture for setting  language overriden by DOTNET_CLI_UI_LANGUAGE:

https://cs.github.com/dotnet/sdk/blob/8a2e6b3f4539f15886455cc21c106d610b3c91ba/src/Cli/dotnet/UILanguageOverride.cs#L27

Reply:
> Should we be using `CurrentCulture` or `CurrentUICulture`? I've never understood when to use which 😕
> 
> edit: although ultimately it doesn't matter because we never format dates, currency, numbers, etc in our string.format calls anyway 🤷

Quickly reading about it, my loose interpretation is `CurrentCulture` is for symbols like numbers, currency, calendars, etc,; and `CurrentUICulture` almost sounds like localized strings in GUI's. So if you prefer English, but a specific region's currency, you can configure those independently since Windows 2000... 👀

Reply:
On second look, the repos I'm seeing haven't been touched in a while. @dominoFire the only reason I can think of is for validation/exception handling I mentioned.